src/runtime/agas/addressing_service.cpp

```diff
@@ -2257,7 +2257,7 @@ namespace detail
                 "request 'symbol_ns_on_event' failed");
             return hpx::future<hpx::id_type>();
         }
-        return result_f;
+        return std::move(result_f);
```

Comment:
Please add a comment stating that this was done to silence Intel13. Otherwise somebody will just remove the `std::move()`.